jboss/container/wildfly/launch/datasources/added/launch/datasource-common.sh

```diff
@@ -31,8 +31,8 @@ function clearDatasourceEnv() {
   local prefix=$1
   local service=$2
 
-  unset ${service}_HOST
-  unset ${service}_PORT
+  unset ${service}_SERVICE_HOST
+  unset ${service}_SERVICE_PORT
```

Comment:
The change should be harmless, those variables are set by Kubernetes when the services are configured by the templates. In general, I think the env files would expect to see them available. However, I've just seen the same pattern in [other files](https://github.com/wildfly/wildfly-cekit-modules/blob/master/jboss/container/wildfly/launch/messaging/added/launch/messaging.sh#L37) being in use and seems to be the initial intention of clearing ${service}_PORT and ${service}_HOST. So, it looks good to me.